assets/server/codes/bulk-issue.html

```diff
@@ -24,17 +24,10 @@
     <div class="card mb-3 shadow-sm">
       <div class="card-header">{{t $.locale "codes.bulk-issue.header"}}</div>
       <div class="card-body">
-        {{if not $hasSMSConfig}}
```

Comment:
is there a replacement to this 'why' warning for disabled?

Reply:
Just kidding, I see that it's now in the flash.

Reply:
Yea, I wanted to try and use the flash to make the style consistent.